packages/editor/src/hooks/use-editor.ts

```diff
@@ -37,6 +47,21 @@ export const useEditor = (
   const forceUpdate = useForceUpdate();
   const editorRef = useRef<TiptapEditor>(editor);
 
+  // TODO: remove
```

Comment:
Remove when?

Reply:
@01zulfi what about this? It looks useless to me.

Reply:
Yes, I need to just verify all the tiptap shortcuts one more time and then I'll remove it.